toolchain/semantics/semantics_node_kind.def

```diff
@@ -84,6 +88,8 @@ CARBON_SEMANTICS_NODE_KIND_IMPL(TupleIndex, "tuple_index", Typed, NotTerminator)
 CARBON_SEMANTICS_NODE_KIND_IMPL(TupleType, "tuple_type", Typed, NotTerminator)
 CARBON_SEMANTICS_NODE_KIND_IMPL(TupleValue, "tuple_value", Typed, NotTerminator)
 CARBON_SEMANTICS_NODE_KIND_IMPL(UnaryOperatorNot, "not", Typed, NotTerminator)
+CARBON_SEMANTICS_NODE_KIND_IMPL(ValueBinding, "value_binding", Typed,
+                                NotTerminator)
```

Comment:
Thoughts on a more verb-like naming of `bind_value`? Or `bind_as_value`?

Reply:
Done.